src/agent/internals/src/test/java/com/google/devtools/cdbg/debuglets/java/BUILD

```diff
@@ -119,3 +166,32 @@ java_library(
     ],
 )
 
+# Include source files to automatically find source location by special comment
+# tag in a source file.
```

Comment:
Make it more explicit as to when this dependency is required.

Reply:
I've updated the comment and target name.